### PR TITLE
Minutes of the 2022-08-25 TC Meeting

### PR DESCRIPTION
**Pull request by sthagen**
Transcribed from notes provided by Thanassis, thanks.

### CODE REVIEW

File: meeting_minutes/220825_SRAIF_TC_65.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+# 1. Opening Activities
```
Comment (pavel-mikula-sonarsource) [inline meeting_minutes/220825_SRAIF_TC_65.md:1]: PR title should have 22-08-25, same as the file name fix

File: meeting_minutes/220825_SRAIF_TC_65.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair Keaton)
+
+- Scgeduled Teleconferences (Thursdays at 08:00 PT / 15:00 UTC for 1.5 hours)
```
Comment (katrinaoneil) [inline meeting_minutes/220825_SRAIF_TC_65.md:74]: Scgeduled -> Scheduled

File: meeting_minutes/220825_SRAIF_TC_65.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+* Continuing to monitor for needed change on sarif.info. 
+* Any new companies want to be featured there can contact Mary Martin directly.
+
+## 4.4 Review outcomes of subgroup discussions
+
+* No new subgroup discussions.
+
+## 4.5 Discuss end-to-end results management (including code insights protocol)
+
+4.5 Discuss end-to-end results management
```
Comment (katrinaoneil) [inline meeting_minutes/220825_SRAIF_TC_65.md:118]: remove?

**Review by katrinaoneil — APPROVED** [review]
A couple of minor typos, but other than that looks great!

File: meeting_minutes/220825_SRAIF_TC_65.md (hunk 1 above)
Comment (sthagen) [reply meeting_minutes/220825_SRAIF_TC_65.md:1]: ... same as the event date (I hope we do not relate PR titles to files 😸 )

File: meeting_minutes/220825_SRAIF_TC_65.md (hunk 3 above)
Comment (sthagen) [reply meeting_minutes/220825_SRAIF_TC_65.md:118]: Sure.

File: meeting_minutes/220825_SRAIF_TC_65.md (hunk 2 above)
Comment (sthagen) [reply meeting_minutes/220825_SRAIF_TC_65.md:74]: It reads so much more interesting though ... thanks.

**Review by dmk42 — APPROVED** [review]
Sorry for the delay.

File: meeting_minutes/220825_SRAIF_TC_65.md (hunk 3 above)
Comment (mohmadalabdalui) [reply meeting_minutes/220825_SRAIF_TC_65.md]: meeting_minutes/  #056220825_SRAIF_TC_65

**Review by mohmadalabdalui — COMMENTED** [review]
#meeting_minutes/220825_SRAIF_TC_65.md

**Pull request merged**